src/components/app/ProfileFilterNavigator.js

```diff
@@ -12,11 +12,12 @@ import {
   getPreviewSelection,
   getProfileFilterPageData,
   getProfileRootRange,
-} from 'firefox-profiler/selectors/profile';
-import { getCommittedRangeLabels } from 'firefox-profiler/selectors/url-state';
```

Comment:
So it turns out we are actually moving to use absolute imports: `firefox-profiler/selectors/url-state` 

See https://github.com/firefox-devtools/profiler/issues/2865

Please revert these back to the absolute path style.

Reply:
Thank you @gregtatum. Made the changes.

Reply:
And sorry about that